### ISSUE TITLE
Accessibility bug in the Select (Grouped checkbox input with filtering) can not be navigated with the arrow keys and component doesn't move cursor to search input on click

### ISSUE THREAD

**Issue by sg00dwin** [patternfly/patternfly-react#5847]
**Describe the issue. What is the expected and unexpected behavior?**
The `<Select>` component [Grouped checkbox input with filtering](https://www.patternfly.org/v4/components/select/#grouped-checkbox-input-with-filtering) does not insert the cursor in the `input` field on click. And thus keyboard accessibility is broken, cannot be navigated by the arrow keys. 

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

https://codesandbox.io/s/bi6pb
Click on "Filter by status". Cursor focus should automatically be within filter input, allowing the user to start typing to filter list or use the arrow keys to traverse list.

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug

**What is your product and what release version are you targeting?**
OpenShift switched to use the `<Select>` component recently https://github.com/openshift/console/pull/8877 
version 4.8

**Comment by sg00dwin** [comment]
![select-arrow-bug](https://user-images.githubusercontent.com/1874151/119857392-84364400-bee1-11eb-98a5-ae3464bf354a.gif)

**Comment by sg00dwin** [comment]
![Uploading select-arrow-bug2.gif…]()

**Comment by sg00dwin** [comment]
This will be fixed in https://github.com/patternfly/patternfly-react/pull/5793 so closing the issue.
